frontend/src/pages/articles/components/articles-list/articles-list.tsx

```diff
@@ -0,0 +1,41 @@
+import { InfiniteScroll } from '~/libs/components/components.js';
+import { getArticleTags } from '~/libs/helpers/helpers.js';
+import { type ArticleWithCountsResponseDto } from '~/packages/articles/articles.js';
+
+import { ArticleCard } from '../components.js';
```

Comment:
As it's the same folder as for articles-list
```suggestion
import { ArticleCard } from '../article-card/article-card.js';
```